Habitica/src/main/java/com/habitrpg/android/habitica/helpers/RemoteConfigManager.java

```diff
@@ -0,0 +1,122 @@
+package com.habitrpg.android.habitica.helpers;
+
+import android.content.Context;
+import android.os.AsyncTask;
+import android.os.Environment;
+import android.util.Log;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.BufferedInputStream;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.net.URL;
+import java.net.URLConnection;
+
+import bolts.Bolts;
+
+/**
+ * Created by keith holliday on 4/7/2017.
+ */
+
+public class RemoteConfigManager {
```

Comment:
I think this should also be cached in SharedPreferences or something like that, so that it's also available when the request might fail.